Tests/Testo/Invoice/Merchant/MerchantServiceTest.php

```
@@ -0,0 +1,190 @@
+        $e2 = $repository->shouldReceive('save');
+        $e2->once()->with($model);
+
+        $service = $this->makeService($repository, $iR);
+        $service->saveMerchant($model, $array);
+
+        Assert::same($inv, $model->getInv());
+        Assert::same(11, $model->reqInvId());
+        Assert::true($model->getSuccessful());
+        $date = $this->dateProperty($model);
+        if (!$date instanceof \DateTime) {
+            throw new \RuntimeException('Expected a DateTime');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '77', column '23'). (undefined)

(MissingImport)